modules/caddyhttp/ip_matchers.go

```diff
@@ -126,11 +115,7 @@ func (MatchRemoteIP) CELLibrary(ctx caddy.Context) (cel.Library, error) {
 
 			for _, input := range strList.([]string) {
 				if input == "forwarded" {
-					if len(m.Ranges) > 0 {
-						return nil, errors.New("if used, 'forwarded' must be first argument")
-					}
-					m.Forwarded = true
-					continue
+					return nil, errors.New("the 'forwarded' option is no longer supported; use the 'client_ip' matcher instead")
```

Comment:
Wonder if we will remember to remove this in a while as well

Reply:
Meh, this is just to give a nicer error message just in case they upgrade from an older version. It can stay forever, it's harmless.